CHANGELOG.md

```diff
@@ -1,8 +1,24 @@
 Changelog
 ------------
 
-##### NEXT
+##### 9.19.0
+
 * Replaced `componentWillMount`, `componentWillReceiveProps`, and `componentWillUpdate` with async-safe lifecycles in advance of React 16.4 deprecation warnings. Added [react-lifecycles-compat](https://github.com/reactjs/react-lifecycles-compat) as a dependency, to ensure backwards compatibility.
```

Comment:
I realize you didn't write this line- (I did)- but can we change "_React 16.4 deprecation warnings_" to "_React 16.x deprecation warnings_". It looks likely that we will not be enabling them with 16.4, but in a subsequent minor release.

Reply:
I see. Will update.